docs/docker-images/buildkit-standalone.md

```diff
@@ -0,0 +1,61 @@
+This image contains `buildkit` with some Earthly-specific setup. This is what Earthly will start when using a local daemon. You can also start it up yourself and use it as a remote/shared buildkit daemon.
+
+## Quickstart
+
+Want to just get started? Here are a couple sample `docker run` commands that cover the most common use-cases:
+
+### Simple Usage (Use Locally)
+
+```bash
+$ docker run --privileged -t -v /tmp/earthly:/tmp/earthly:rw earthly/buildkitd:latest
+```
+
+Heres a quick breakdown:
+
+- `--privileged` is required. This is because `earthly` needs some privileged `buildkit` functionality.
+- `-t` tells Docker to emulate a TTY. This makes the `buildkit` log output colorized.
+- `-v /tmp/earthly:/tmp/earthly:rw` mounts the hosts `/tmp/earthly` into the containers `/tmp/earthly`. This is used as a temporary/working directory for `buildkitd` during builds.
+
+Assuming you are running this on your machine, you could use this `buildkitd` by setting `EARTHLY_BUILDKIT_HOST=docker-container://<container-name>`, or by specifying the appropriate values in `config.yml`.
+
+### Usage (Use As Remote)
+
+```bash
+$ docker run --privileged -t -v /tmp/earthly:/tmp/earthly:rw -e BUILDKIT_TCP_TRANSPORT_ENABLED=true -p 127.0.0.1:8372:8372 earthly/buildkitd:latest
+```
+
+Omitting the options already discussed from the simple example:
+
+- `-e BUILDKIT_TCP_TRANSPORT_ENABLED=true` makes `buildkitd` listen on a TCP port instead of a Unix socket.
+- `-p 127.0.0.1:8372:8372` forwards the hosts port 8372 to the containers port 8372. When using TCP, `buildkit` will always listen on 8372, but you can configure the apparent port by forwarding a different port on your host.
+
+Assuming you ran this on another machine named `fast-builder`, you could use this remote `buildkitd` by setting `EARTHLY_BUILDKIT_HOST=tcp://fast-builder:8372`, or by specifying the address in your `config.yml`.
+
+## Using This Image
+
+### Requirements
+
+#### Privileged Mode
+
+This image needs to be run as a privileged container. This is because `buildkitd` needs appropriate access to start and run additional containers itself via `runc`.
+
+#### External Usage
+
+To use this image externally, it requires you to forward a port on your machine to the containers port 8372. You will need to ensure that external access ti the machine on the port you chose is possible as well.
+
+If you are using this container locally, please note that `EARTHLY_BUILDKIT_HOST` values where the host is `127.0.0.1`, ` ::1/128`, or `localhost` are considered local and will result in Earthly attempting to manage the buildkit container itself. Consider using your hostname, or another alternative name in these cases.
```

Comment:
When I first read this it made me think that EARTHLY_BUILDKIT_HOST is a setting of buildkit given the topic of this page. Maybe clarify this a bit more?